test/bin/stress_testing.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+#
+# This script will enable stress scenarios to run MicroShift tests
+# with constricted resources
+
+set -uo pipefail
+
+SCRIPT_NAME=$(basename "$0")
+IP=$(nslookup "$(hostname)" | grep -v "#" | awk '/Address:/{print $2}')
```

Comment:
Use `hostname -I | awk '{print $1}'` to get the first IP address of the local host.

Reply:
thanks! changed